tests/unit_tests/test_margin_calculation.py

```diff
@@ -6,24 +6,32 @@
 
 def test_margin_binance():
     # using VET-GBP
-    buy_fee = 0.07434422700000001
-    buy_filled = 74.344227
-    buy_price = 0.1249
     buy_size = 595.23
+    buy_fee = 0.59523  # round(buy_size * 0.001, 8)
```

Comment:
The idea is not to change those tests and instead, use it the way it was to make sure your changes don't break anything... It was changed in the last PR because we introduced it there.

Reply:
You are right, I made change on the precisions that's why needed to update test cases.